test/e2e/configuration_anomaly_detection_test.go

```diff
@@ -205,13 +226,16 @@ var _ = Describe("Configuration Anomaly Detection", Ordered, func() {
 			Expect(err).ToNot(HaveOccurred(), "failed to scale down alertmanager")
 			fmt.Printf("Alertmanager scaled down from %d to 0 replicas. Waiting...\n", originalAMReplicas)
 
-			time.Sleep(20 * time.Minute)
+			_, err = testPdClient.CreateRequest("AlertManagerDown", clusterID)
```

Comment:
This test is still for ClusterHasGoneMissing, but we're initiating it with a different misconfiguration.